vivisect/qt/funcgraph.py

```diff
@@ -277,6 +301,31 @@ def _hotkey_deczoom(self):
         #self.vw.vprint("NEW ZOOM    %f" % newzoom)
         self.mem_canvas.setZoomFactor(newzoom)
 
+    def refresh(self):
+        '''
+        Cause the Function Graph to redraw itself.
+        This is particularly helpful because comments and name changes don't
+        immediately display.  Perhaps someday this will update only the blocks
+        that have changed since last update, and be fast, so we can update
+        after every change.  
+        '''
+        self._last_viewpt = self.mem_canvas.page().mainFrame().scrollPosition()
+        self.clearText()
+        self.fva = None
+        self._renderMemory()
+
+    @workthread
```

Comment:
having a thread other than the qt main thread carry this out seems incorrect. did something make this decorator needed? If so, seems like a race.